pycheribuild/projects/cross/eigen.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2020 Alex Richardson
```

Comment:
You probably want your copyright here instead of listing me.

Reply:
Well, you wrote cheribuild and the [whatever].py script on which eigen.py was based.  I feel like my modification of the script is similar to providing a command line argument.